src/redux/reducers/TodoReducer.ts

```diff
@@ -3,18 +3,19 @@ import { AddItemPayload } from '../actions/todoActions/addItemAction';
 import { ItemDeletePayload } from '../actions/todoActions/deleteItemAction';
 import { TodoActionTypes } from '../actions/todoActions';
 import { DoneItemPayload } from '../actions/todoActions/doneItemAction';
-import { SignInPayload } from '../actions/todoActions/signInAction';
+import { AuthActionsTypes } from '../actions/authActions';
+import { SignInPayload } from '../actions/authActions/signInAction';
 
 export type TodoReducerState = {
   todoItems: TodoItemType[];
   userToken: SignInPayload['userToken'] | null;
-  user: SignInPayload['user'] | null;
+  user: SignInPayload['user'];
 };
 
 const initialState: TodoReducerState = {
   todoItems: [],
   userToken: null,
-  user: null,
+  user: undefined,
```

Comment:
have you split your reducer to auth and todo?

Reply:
i fix it